src/lib/compose-cart.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+// $FlowFixMe
+export const removeFromCart = (items, currentItems = []) => {
+    return items.reduce((accumulator, item) => {
+        if (item.quantity === undefined) {
+            return accumulator.filter(curItem => curItem.id !== item.id);
+        }
+  
+        if (typeof item.quantity !== 'number') {
```

Comment:
are we certain quantity is expected to be a `number`?

Reply:
https://github.com/paypal/paypal-muse-components/blob/e3c8056fe741598f4cc82878f7a49efe82e7a7c8/src/tracker-component.js#L25

Reply:
for this..... what if we just made a best effort decision?
```
const quantity = parseInt( item.quantity, 10 ) || 0
```

Reply:
What would the advantage be of accepting 'quantity' as a string?

Reply:
not much.... `removeFromCart( [{title: 'shoes', quantity: '2'}] )` feels strange as a developer. So for consistency, we should either change this `if` statement or change the `type` to number